packages/eslint-plugin/tests/rules/consistent-type-imports.test.ts

```diff
@@ -1215,5 +1222,24 @@ const a: Default = '';
         },
       ],
     },
+    {
```

Comment:
Could you also please add tests for:

- default imports
- namespace imports
- mixed default and named imports
- mixed default and namespace imports

Reply:
Yeah I'm not really familiar with the whole approach here - actually it would really help if someone with experience could take this over.
Do you perhaps have snippets for which kind of test cases you would like to see added?

Reply:
Simply put - you should cover as many cases as you can think of to ensure that everything is working as expected.

More test cases also document how you expect the option to behave, to aid future contributors.

A single test case does not cover or explain much, which means it's possible the option does not work as you intend, and it's easy for future contributors to introduce regressions.

Reply:
Added more cases - most of the other ones should also not be necessary I think as the reminder just tests various types of imports, but the different variations of (types-only, mixed, value-only) are all covered in the tests.